### PR TITLE
[13.0][MIG] document_page_environmental_aspect

### PR DESCRIPTION
**Pull request by flachica**
@NachoAlesLopez @Guadaltech @max3903 Review, please

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: document_page_environmental_aspect/__manifest__.py
Comment (ivantodorovich) [inline document_page_environmental_aspect/__manifest__.py:20]: You can use the new header according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1

```
# Copyright 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

File: document_page_environmental_aspect/__manifest__.py
Comment (ivantodorovich) [inline document_page_environmental_aspect/__manifest__.py]:
```suggestion
    "version": "13.0.1.0.0",
```

**Review by ivantodorovich — APPROVED** [review]
Minor comments :+1: 

I'm approving in advance

**Comment by ivantodorovich** [comment]
It would be good to squash translation commits ,to clean the history:

there are many "Launchpad automatic translations update." and OCA Transbot commits

**Comment by flachica** [comment]
> It would be good to squash translation commits ,to clean the history:
> 
> there are many "Launchpad automatic translations update." and OCA Transbot commits

I still have not understood how to make a squash for OCA. The guide I have seen does not clarify this matter for me. I just made the suggested changes to the code but I think the commit messages remain the same

**Comment by ivantodorovich** [comment]
Hmmm. Now we have duplicate commits in the history.
Maybe you tried rebasing, and then pushed without `--force` ?

image


As for squashing, there are many guides online on how to squash commits
ie: https://www.internalpointers.com/post/squash-commits-into-one-git

Anyways, it's not that important. But we do need to fix the commit history because now it's duplicated ^^ I'm hoping a `git push --force` on your side solves the issue :)

**Comment by flachica** [comment]
> Hmmm. Now we have duplicate commits in the history.
> Maybe you tried rebasing, and then pushed without `--force` ?
> 
> image
> 
> As for squashing, there are many guides online on how to squash commits
> ie: https://www.internalpointers.com/post/squash-commits-into-one-git
> 
> Anyways, it's not that important. But we do need to fix the commit history because now it's duplicated ^^ I'm hoping a `git push --force` on your side solves the issue :)

If you solve this in github I will try in next PR, Ok?

**Comment by ivantodorovich** [comment]
I can't solve this in github :sweat: it has to be solved on your branch before merging

**Comment by flachica** [comment]
I have tried to clear the history. You can check @ivantodorovich? Thank you very much for your patience.

**Comment by flachica** [comment]
I think this history is not correct. I think the best thing to do at this point is to create a new PR with the changes. Okay? @ivantodorovich

**Comment by ivantodorovich** [comment]
Hi @flachica 
There are no duplicates now but it looks some commits are missing, maybe you squashed them together ?

Yes I agree at this point it might be better to open a new PR :sweat: 
Sorry for this and thanks for taking the time to work on it

**Comment by flachica** [comment]
No problem, friend. I open the new PR ASAP. You just have to be aware because the number will change in the comments and it can be a bit messy. Sorry for the inconvenience, but I am getting all the tools.

**Comment by flachica** [comment]
New PR on #315

**Pull request closed without merging**
